src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -259,10 +259,18 @@ void GcsActorManager::HandleCreateActor(const rpc::CreateActorRequest &request,
       request,
       [reply, send_reply_callback, actor_id](const std::shared_ptr<gcs::GcsActor> &actor,
```

Comment:
Can you rewrite the docstring of `CreateActor` method? (the definition of callback param). It is incorrect with the new implementation. (it says the callback is called when the actor is created successfully )